OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -265,6 +281,37 @@ void SetupGameModeDropdown(MapClassification tab, DropDownButtonWidget gameModeD
 			}
 		}
 
+		void SetupOrderByDropdown(MapClassification tab, ScrollItemWidget itemTemplate)
+		{
+			if (orderByDropdown == null)
+				return;
+
+			var orderByDict = new Dictionary<string, Func<MapPreview, long>>()
+			{
+				{ orderByPlayer, m => m.PlayerCount },
+				{ orderByDate, m => -m.ModifiedDate.Ticks }
+			};
+
+			if (orderByFunc == null)
+				orderByFunc = orderByDict[orderByPlayer];
+
+			Func<string, ScrollItemWidget, ScrollItemWidget> setupItem = (o, template) =>
+			{
+				var item = ScrollItemWidget.Setup(template,
+					() => orderByFunc == orderByDict[o],
+					() => { orderByFunc = orderByDict[o]; EnumerateMaps(tab, itemTemplate); });
```

Comment:
Now that the function has been moved this is incorrect

```suggestion
					() => { orderByFunc = orderByDict[o]; EnumerateMaps(currentTab, itemTemplate); });
```